device/mellanox/x86_64-nvidia_sn2201-r0/system_health_monitoring_config.json

```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/system_health_monitoring_config.json
```

Comment:
please check path. might be that you need extra ../ as in other links

Reply:
I verified and the path is valid.